### ISSUE TITLE
[Bug] Voice over not working on video player renderer

### ISSUE THREAD

**Issue by Sigve84** [xamarin/Xamarin.Forms#9832]
### Description
It is not possible to select elements (play, pause etc.) on avplayer when voiceover is activated in Xamarin Forms apps. This works fine in Xamarin ios native. 
### Steps to Reproduce

1. Download example code from this url : https://github.com/Sigve84/xamarinformsvideoplayer
2. Activate voice over on IOS and start Xamarin forms app

### Expected Behavior
Be able to select elements in player when voice over is activated
### Actual Behavior
Cannot select elements, and voice over is not able to read out element names
### Reproduction Link

https://github.com/Sigve84/xamarinformsvideoplayer

**Comment by PureWeen** [comment]
My apologies @Sigve84 but this is going to be out of scope for us to fix on XF

If you haven't already, I also recommend updating to the latest version Xamarin.Forms or at least to Xamarin.Forms 5.0.0.2083, which has many accessibility improvements. These improvements include an experimental accessibility flag that addresses a lot of improvements in Android accessibility.

To leverage this flag on the latest versions of Xamarin Forms, the following code can be inserted in the App.xaml.cs file, following the InitializeComponent call and before the MainPage is set: Xamarin.Forms.Device.SetFlags(new List<string> { "Accessibility_Experimental"});
